css/styles.css

```diff
@@ -0,0 +1,85 @@
+* {
+    box-sizing: border-box;
+}
+
+html {
+    font-size: 62.5%;
+}
+
+body {
+    font-family: 'Play', sans-serif;
+    color: #ffffff;
+}
+
+#splash-screen {
+    height: 800px;
+    width: 1000px;
+    margin: auto;
+    overflow: hidden;
+    background-image: url("../img/background.jpg");
+    background-size: cover;
+}
+
+#splash-rocket-img {
+    width: 724px;
+    height: 232px;
+}
+
+#splash-title-rocket {
+    position: relative;
+    right: 1224px;
+    top: 50px;
+    width: 2000px;
+}
+
+#splash-flames {
+    height: 232px;
+    width: 500px;
+}
+
+#splash-title {
+    position: absolute;
+    text-align: center;
+    font-size: 9.6rem;
+    width: 1000px;
+    margin-top: 100px;
+    display: none;
+}
+
+#splash-open-game {
+    display: block;
+    width: 200px;
+    padding: 10px;
+    margin: 50px auto 50px auto;
+    font-size: 3.2rem;
+    text-align: center;
+    border-style: solid;
+    text-decoration: none;
+    color: #ffffff;
+    position: relative;
```

Comment:
not clear this is clickable, give it a `cursor: pointer;`

Reply:
and maybe a hover state?

Reply:
Fixed in eda5d1e